app/src/main/java/erevacation/com/userlistanko/homescreen/profile/ProfileFragment.kt

```diff
@@ -0,0 +1,50 @@
+package erevacation.com.userlistanko.homescreen.profile
+
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import erevacation.com.userlistanko.basic.BasicFragment
+import erevacation.com.userlistanko.homescreen.HomeActivity
+import org.jetbrains.anko.AnkoContext
+
+class ProfileFragment : BasicFragment<ProfileContract.ProfilePresenter>(), ProfileContract.ProfileView {
+
+    val ankoFragmentProfileUI: AnkoFragmentProfileUI = AnkoFragmentProfileUI()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?)
+            : View = ankoFragmentProfileUI.createView(AnkoContext.Companion.create(activity as HomeActivity, this))
+
+    var name: String = ""
+    var surname: String = ""
+    var image: String = ""
+    var profileInfoList: ArrayList<String> = arrayListOf()
```

Comment:
Can we make all of this `var` private ?

Reply:
than we cant access them in ProfileFragmentPresenter